payments-ui-core/src/test/java/com/stripe/android/ui/core/elements/AuBankAccountNumberConfigTest.kt

```diff
@@ -0,0 +1,32 @@
+package com.stripe.android.ui.core.elements
+
+import com.google.common.truth.Truth
+import org.junit.Test
+
+class AuBankAccountNumberConfigTest {
+    private val auBankAccountNumberConfig = AuBankAccountNumberConfig()
+
+    @Test
+    fun `verify blank bank account number returns blank state`() {
+        Truth.assertThat(auBankAccountNumberConfig.determineState(""))
+            .isEqualTo(TextFieldStateConstants.Error.Blank)
+    }
+
+    @Test
+    fun `verify incomplete bank account number is in incomplete state`() {
+        Truth.assertThat(auBankAccountNumberConfig.determineState("123"))
+            .isInstanceOf(TextFieldStateConstants.Error.Incomplete::class.java)
+
+        Truth.assertThat(auBankAccountNumberConfig.determineState("123456"))
+            .isInstanceOf(TextFieldStateConstants.Error.Incomplete::class.java)
+
+        Truth.assertThat(auBankAccountNumberConfig.determineState("1234567890"))
```

Comment:
I think in the case of too many characters TextFieldStateConstants.Error.Invalid would be a better error case.  Incomplete is intended more for the case where either the expected number of characters are missing, or it is missing an expected character like '@' in the case of an email.